docs/source/integrations/burrow_example.rst

```diff
@@ -150,85 +150,120 @@ Note the last line of the send function that emits a Sent event which gets recor
 	emit Sent(msg.sender, receiver, amount);
```

Comment:
BTW. please in line 130 make comments in one column, so instead of:
```
	params = ("40c10f19”                                                             # selector
	          "000000000000000000000000f205c4a929072dd6e7fc081c2a78dbc79c76070b"  # address
	          "00000000000000000000000000000000000000000000000000000000000003e8"  # amount
	         )
```
please:
```
	params = ("40c10f19”                                                          # selector
	          "000000000000000000000000f205c4a929072dd6e7fc081c2a78dbc79c76070b"  # address
	          "00000000000000000000000000000000000000000000000000000000000003e8"  # amount
	         )
```